src/scribe_data/resources/language_meta_data.json

```diff
@@ -15,56 +16,64 @@
       "iso": "en",
       "qid": "Q1860",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
+      "ignore-words": [],
+      "word-types": ["nouns", "verbs", "translated_words"]
     },
     {
       "language": "french",
       "iso": "fr",
       "qid": "Q150",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": ["XXe"]
+      "ignore-words": ["XXe"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "german",
       "iso": "de",
       "qid": "Q188",
       "remove-words": ["of", "the", "The", "and", "NeinJa", "et", "redirect"],
-      "ignore-words": ["Gemeinde", "Familienname"]
+      "ignore-words": ["Gemeinde", "Familienname"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "italian",
       "iso": "it",
       "qid": "Q652",
       "remove-words": ["of", "the", "The", "and", "text", "from"],
-      "ignore-words": ["The", "ATP"]
+      "ignore-words": ["The", "ATP"],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "portuguese",
       "iso": "pt",
       "qid": "Q5146",
       "remove-words": ["of", "the", "The", "and", "jbutadptflora"],
-      "ignore-words": []
+      "ignore-words": [],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "russian",
       "iso": "ru",
       "qid": "Q7737",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
+      "ignore-words": [],
+      "word-types": ["nouns", "verbs", "translated_words", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
     },
     {
       "language": "spanish",
       "iso": "es",
       "qid": "Q1321",
       "remove-words": ["of", "the", "The", "and"],
-      "ignore-words": []
+      "ignore-words": [],
+      "word-types": ["nouns", "verbs", "translations", "emoji_keywords", "prepositions", "autosuggestions"]
```

Comment:
Also looking into this, @mhmohona, it looks like you need to install an autoformatter as this would normally be taken care of. The one that I use is [ruff](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff) :)

Reply:
Note that I removed the word-types keys given that we're now using the directory structure to derive what can be queried 😊